### DISCUSSION TITLE
feature request - Warm up time

### DISCUSSION THREAD

**Discussion by Mehrdadg** [jgyates/genmon#1033] [Ideas]
the current Version V1.19.01 shows Startup delay parameter in the outage tab for my 48KW Liquid cooled generator and it would be nice if the SW could also report the Warm up time settings of the generator.

additionally, it would be nice if one could remotely adjust both the Startup delay as well as Warm up time. Is there some safety issue as why this is not included?

thanks

**Comment by jgyates** [comment]
@Mehrdadg 

I will add this to the feature list. I have not added this to date as until recently the code did not support an adequate method to write values from the web interface that input parameters. I will update this thread once the changes are complete.

**Comment by jgyates** [comment]
https://github.com/jgyates/genmon/blob/master/feature_requests.md

**Comment by jgyates** [comment]
I added the following for Evolution Liquid Cooled controllers:

Set Exercise Duration
Set Start Up Delay
Set Warm Up Delay
Set Threshold Volts
Set Pickup Volts

These are at the bottom of the Maintenance page. Let me know if you have any issues or feedback.

These are only available on Liquid Cooled Evolution as the other Evolution/Nexus variants do not expose this functionality via modbus.

**Reply by Mehrdadg** [reply]
I tried the settings and it does as advertised.
Many thanks for adding these features in such a short time.
Regards
